ARM-wvd-templates/AddVirtualMachinesToHostPool/AddVirtualMachinesTemplate.json

```diff
@@ -68,6 +68,20 @@
                 "description": "The password that corresponds to the existing domain username."
             }
         },
+        "vmAdministratorAccountUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "A username to be used as the virtual machine administrator account. The domain administrator username will be used if this parameter is not provided."

```

Comment:
I think this and the description for vmAdministratorAccountPassword should specify that both vmAdministratorAccountUsername and vmAdministratorAccountPassword must have a value for them to be used, since otherwise a user might fill in one but not the other and expect it to work.
I think it would also be helpful to specify the exact parameter name that will be used if this doesn't have a value, such as administratorAccountUsername in this case.

Reply:
@lintFan I have modified the "vmadministratorAccountUserName" description to `"A username to be used as the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by administratorAccountUsername and administratorAccountPassword will be used."`

Similarly for "vmAdministratorAccountPassword" description to `"The password associated with the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by administratorAccountUsername and administratorAccountPassword will be used."`